lib/emscripten/src/lib.rs

```diff
@@ -770,7 +730,15 @@ pub fn generate_emscripten_env(globals: &mut EmscriptenGlobals) -> ImportObject
         "asm2wasm" => {
             "f64-rem" => func!(crate::math::f64_rem),
         },
-    }
+    };
+
+    import_object
+}
+
+pub fn nullfunc_x(ctx: &mut Ctx, _x: u32) {
```

Comment:
Should be just `nullfunc`? (without an x)

```
pub fn nullfunc_x(ctx: &mut Ctx) {
...
}
```

Reply:
sure, sound good